hashicorp/web-unified-docs
# Chore: remove Sentinel content

# Changes

- Remove sentinel content from code base to prep for education engineer test. This includes:
  - Removing sentinel from the `content` dir
  - Removing sentinel from the `public/assets` dir

- [Asana task](https://app.asana.com/0/1207899860738460/1208561647881341/f)

# Testing

1. Make a curl request or use Postman to send a `GET` request 
```
curl https://web-presence-experimental-docs.vercel.app/api/content/sentinel/nav-data/latest/docs
```
1. You should receive a valid response, such as this:
<img width="1145" alt="Screenshot 2024-10-16 at 16 15 31" src="https://github.com/user-attachments/assets/d8ef1347-a878-4f66-9493-8b50141cf14b">


1. Make a curl request or use Postman to send a `GET` request
```
curl https://web-presence-experimental-docs-git-heat-chorer-3888a2-hashicorp.vercel.app/api/content/sentinel/nav-data/latest/docs
```
1. You should receive a response of `Not Found`
